Neo4j.Driver/Neo4j.Driver/Internal/IO/BoltReader.cs

```diff
@@ -81,14 +81,22 @@ public Task ReadAsync(IMessageResponseHandler responseHandler)
                 _chunkReader.ReadNextMessageAsync(_bufferStream)
                     .ContinueWith(t =>
                     {
-                        ProcessMessage(responseHandler);
+                        ConsumeMessages(responseHandler);
                     }, TaskContinuationOptions.ExecuteSynchronously);
         }
 
-        private void ProcessMessage(IMessageResponseHandler responseHandler)
+        private void ConsumeMessages(IMessageResponseHandler responseHandler)
         {
             _bufferStream.Position = 0;
 
+            while (_bufferStream.Length > _bufferStream.Position)
```

Comment:
I see, then I totally misunderstood this code. I will merge this when the build finished.